src/errorHandler.ts

```diff
@@ -100,7 +114,7 @@ export function safeExecution(
  */
 export function setupErrorHandler(): Disposable {
     // Setup the statusBarItem
-    statusBarItem = window.createStatusBarItem();
+    statusBarItem = window.createStatusBarItem(StatusBarAlignment.Right, -1);
     statusBarItem.text = 'Prettier';
     statusBarItem.show();
```

Comment:
Hmm, that's not that easy to do because we want to return the disposable command. So I added the `.hide()` instead of `.show()` thingy